docs/user/alerting/alerting-setup.asciidoc

```diff
@@ -20,6 +20,9 @@ If you are using an *on-premises* Elastic Stack deployment with <<using-kibana-w
 * You must enable Transport Layer Security (TLS) for communication <<configuring-tls-kib-es, between {es} and {kib}>>. {kib} alerting uses <<api-keys, API keys>> to secure background rule checks and actions, and API keys require {ref}/configuring-tls.html#tls-http[TLS on the HTTP interface]. A proxy will not suffice.
 * If you have enabled TLS and are still unable to access Alerting, ensure that you have not {ref}/security-settings.html#api-key-service-settings[explicitly disabled API keys].
 
+Allow expensive queries `search.allow_expensive_queries` should be set to `true`. By default the value is `true`.
+This is the requirenment, which comes from the Alerting framework needs to retrive the actions references (using joining query) and building telemetry queries using `scripts`. According to the {ref}/query-dsl-script-query.html#_allow_expensive_queries_4[documentation] the scripts won't work if `search.allow_expensive_queries` is `false`. 
```

Comment:
Also, feels a bit wordy overall. I'd be curious what @elastic/kibana-docs thinks here

Reply:
Ya, I don't think we need to explain exactly why we need this on.  I'd say we could change the final paragraph to just say something like "the Alerting framework uses queries which require this setting to be `true`".

Reply:
++ to the sentence that @pmuellr suggests, with one change"

The Alerting framework uses queries that require this setting to be `true`.

Also, the title of this doc should be sentence case:  Alerting set up

Reply:
Fixed. Just left the last sentence about the scripts docs.